### ISSUE TITLE
TypeError: _cast_by_example

### ISSUE THREAD

**Issue by delirious-lettuce** [guardicore/monkey#109]
## Expected Behavior
Not raise a TypeError

## Actual Behavior
Raises a TypeError

## Steps to Reproduce the Problem

  1. Call `_cast_by_example` with a `value` which isn't `None` and `example` as a tuple with a minimum length of `1`.

```python
>>> _cast_by_example('value', ('example',))
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "<input>", line 14, in _cast_by_example
TypeError: 'NoneType' object is not iterable
```

The problem is in this line:

https://github.com/guardicore/monkey/blob/8f0251e822f937d9ee38bccc982fb50e8a9fcdb2/infection_monkey/config.py#L28

The issue is calling `tuple(None)`:

```python
>>> tuple(None)
Traceback (most recent call last):
  File "<input>", line 1, in <module>
TypeError: 'NoneType' object is not iterable
```

## Potential Fix

The problem line is almost exactly the same as this line (four lines below the problem line):

https://github.com/guardicore/monkey/blob/8f0251e822f937d9ee38bccc982fb50e8a9fcdb2/infection_monkey/config.py#L32

After looking at that, it seems like a potential fix would be either:

```python
if value is None or value == (None,):
# or
if value is None or value == tuple([None]):
```

Both of these produce a tuple with one element which is `None`.
```python
>>> (None,)
(None,)
>>> tuple([None])
(None,)
```

I didn't send a PR since I wasn't exactly sure if this would be correct and if so, which one of these would be preferred.

**Comment by danielguardicore** [comment]
Hey,
Well, that's a bug. I think you're right on the solution. Want to open a one liner PR? 🥇 

Also, what did you change in the configuration to run into this bug?

**Comment by danielguardicore** [comment]
Will be solved in a bugfix soon :)

**Comment by danielguardicore** [comment]
Should be fixed in #111
